README.md

```diff
@@ -43,6 +43,7 @@ main() async {
     await sentry.captureException(
       exception: error,
       stackTrace: stackTrace,
+      extra: additionalCustomData, // Map<String, dynamic> | must be JSON-serializable
```

Comment:
I'd try to avoid adding extra as an overload to `captureException` so the API doesn't differ from the other Sentry SDKs ([documented here](https://develop.sentry.dev/sdk/unified-api/)).

Going further we'd like to add [`Scope`s](https://develop.sentry.dev/sdk/unified-api/#scope) and a way to add data to `captureException` would be either by providing a instance of `Scope` or accessing the clone of the current scope via a callback.

We'll be adding such APIs and more in the upcoming months.